docs/releases/2.1.txt

```diff
@@ -180,6 +180,70 @@ Models
 
 * Query expressions can now be negated using a minus sign.
 
+* The new :class:`~django.db.models.functions.Abs` database function returns
```

Comment:
In this case, I think would be better to say something like "The new math database functions ... (some general)" rather than calling out every function here.